event/src/test/java/com/bazaarvoice/emodb/event/db/astyanax/DefaultSlabAllocatorTest.java

```diff
@@ -0,0 +1,61 @@
+package com.bazaarvoice.emodb.event.db.astyanax;
+
+import com.bazaarvoice.emodb.common.dropwizard.lifecycle.LifeCycleRegistry;
+import com.codahale.metrics.MetricRegistry;
+import com.google.common.collect.Iterators;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.mockito.Mockito.mock;
+import static org.testng.Assert.assertTrue;
+
+public class DefaultSlabAllocatorTest {
+
+    @Test
+    public void allocateGreaterThanMaxEvents() {
+        List<Integer> sizes = new ArrayList<Integer>();
+        for (int i=0; i<Constants.MAX_SLAB_SIZE*3/2; i++) {
+            sizes.add(64);
+        }
+        DefaultSlabAllocator slabAllocator = new DefaultSlabAllocator(mock(LifeCycleRegistry.class), mock(ManifestPersister.class), "metrics", mock(MetricRegistry.class));
+        try {
+            slabAllocator.allocate("mychannel", Constants.MAX_SLAB_SIZE*3/2, Iterators.peekingIterator(sizes.iterator()));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "mychannel" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS1192)